static/src/js/components/MetadataForm/MetadataForm.jsx

```diff
@@ -138,6 +140,18 @@ const MetadataForm = () => {
 
   useEffect(() => {
     const { draft: fetchedDraft } = data || {}
+    const { revisionId: cmrRetrievedRevisionId } = fetchedDraft || ''
+
+    if (cmrRetrievedRevisionId && (cmrRetrievedRevisionId !== appContextRevisionId)) {
+      addNotification({
```

Comment:
I don't think this should be a warning, they are editing a document that isn't the latest revision.

Reply:
Changed.